src/client/api/event.c

```diff
@@ -59,6 +59,12 @@ static unsigned int eq_ref;
  */
 static tse_sched_t daos_sched_g;
 
+void
+daos_eq_ref_set(int value)
+{
+    eq_ref = value;
+}
```

Comment:
what purpose does this serve? why do we have to set eq_ref value at all?

Reply:
Without it, we fail on this assertion:
```
self_test: src/client/api/event.c:163: daos_get_crt_ctx: Assertion `eq_ref > 0' failed.
```
Originally I used `daos_eq_lib_init()`, but that seems to have unwanted side effects (e.g., calls `crt_init()`, creates contexts, etc.)